src/inference/dev_api/ie_icore.hpp

```diff
@@ -60,6 +61,22 @@ class ICore {
                                                     const std::string& deviceName,
                                                     const std::map<std::string, std::string>& config = {}) = 0;
 
+    /**
+     * @brief Creates an executable network from a network object.
+     *
+     * Users can create as many networks as they need and use
+     *        them simultaneously (up to the limitation of the hardware resources)
+     *
+     * @param network CNNNetwork object acquired from Core::ReadNetwork
+     * @param remoteCtx  "Remote" (non-CPU) accelerator device-specific execution context to use
+     * @param config Optional map of pairs: (config parameter name, config parameter value) relevant only for this load
+     * operation
+     * @return An executable network reference
+     */
+    virtual SoExecutableNetworkInternal LoadNetwork(const CNNNetwork& network,
```

Comment:
Why do we need this new method?

Reply:
yes, the plugins (and Auto-BAtching) use the Core thru the IECore, so the method (LoadNetwork with RemoteConetxt) is not new, it just exposes the Core functionality to plugins:https://github.com/openvinotoolkit/openvino/pull/7883/files#diff-f4dddf68fff38766cf43b5c758e47e1e1f9e3d395dc5cea2048c662935fc82d3R1376
same for CreateContext below